index.html

```diff
@@ -0,0 +1,258 @@
+            <h2 class="card d-flex">
+                <img src="Snapshoot Portfolio 4.png" alt="Card Image">
+                <h2 class="works-primary-text"
+                >Multi-Post <br/> Stories 
+                </h2>
+                <ul class="card-client-list d-flex">
+                    <li><h3>CANOPY</h3></li>
+                    <li><h2 class="dot"></h2></li>
+                    <li><h4>Back End Dev</h4></li>
+                    <li><h2 class="dot"></h2></li>
+                    <li><h4>2015</h4></li>
+                </ul>
+                <p class="card-primarytext">
+                    A daily selection of privately personalized reads; no accounts or sign-ups required.
+                </p>
+                <ul class="tags d-flex">
+                    <li>HTML</li>
+                    <li>css</li>
+                    <li>javaScript</li>
+                </ul> 
+                <button class="button-see-project" type="button">See Project</button>
+            </h2>
+        </h2>
+    </h2>
+</section>
+<section class="about app-bckground-color d-flex">
+    <h2 class="about-myself app-margin">
+        <h2 class="about-primary-text-0"
+        >About myself 
+        </h2>      
+        <p class="about-primary-text-2">
+            Hello I’m a software developer! I can help you build a product, feature, or website. Look through some of my work and experience! If you like what you see and have a project you need coded, don't hesitate to contact me.
+        </p>
+        <h3 class="headline-letsconnect">
+            LET’S CONNECT
+        </h3>
+        <ul class="about-myself-social-media-icons">
+            <li><a href="#"><img src="twitter-icon.png" alt="Twitter"></a></li>
+            <li><a href="#"><img src="Linkedin-icon.png" alt="LinkedIn"></a></li>
+            <li><a href="#"><img src="mail-icon.png" alt="Email"></a></li>
+            <li><a href="#"><img src="git-icon.png" alt="GitHub"></a></li>
+            <li><a href="#"><img src="chill-icon.png" alt="Chill"></a></li>
+        </ul>
+        <div><button class="button-see-project-2" type="button">Get My Resume</button></div>
+        <div class="about-body app-margin">
+            <h2>Languages</h2>
+            <ul class="technologies">
+                <li>
+                    <img src="Ellipse 1.png" alt="Icon">
+                    <p>JavaScript</p>
+                </li>
+                <li>
+                    <img src="Ellipse 2.png" alt="Icon">
+                    <p>HTML</p>
+                </li>
+                <li>
+                    <img src="Ellipse 3.png" alt="Icon">
+                    <p>CSS</p>
+                </li>
+            </ul>
+        </div>
+  <li class="resume-list frameworks">
+    <div class="lang-container">
+      <details>
+        <summary>
+          Frameworks
+        </summary>
+        <ul class="lang-list">
+          <li class="skill-item">
+            N/A
+          </li>
+        </ul>
```

Comment:
- [ ] As suggested by the previous reviewer, kindly stick to the Figma design as much as possible, Currently in your design the drop-down icon is at the back of the text but if you check closely in the Figma design the icon is in front of all the text i.e `Languages`, `Frameworks` and `Skills`. Kindly check the Figma design and be sure you implement all the necessary design. Thank you